tests/unit/disco/sagas/testDiscovery.js

```diff
@@ -0,0 +1,104 @@
+import SagaTester from 'redux-saga-tester';
```

Comment:
A comment on this file name: we tend to use `Disco` everywhere so calling this `testDiscovery` rather than `testDisco` stood out to me. Of course, "disco" is a weird code name/shorthand itself, so up to you. Just figured I'd point it out.